homeassistant/helpers/trigger.py

```diff
@@ -97,10 +96,30 @@ def starts_with_dot(key: str) -> str:
     return key
 
 
+def get_absolute_trigger_key(domain: str, key: str) -> str:
```

Comment:
```suggestion
def _get_absolute_trigger_key(domain: str, key: str) -> str:
```
this should be internal